src/CurrentAndNextFeeSchedule.js

```diff
@@ -0,0 +1,62 @@
+import * as proto from "@hashgraph/proto";
+export default class CurrentAndNextFeeSchedule {
```

Comment:
This type should be `FeeSchedules` instead of `CurrentAndNextFeeSchedule`

Reply:
do you mean name?

Reply:
also would that mean 
import FeeSchedule = proto.CurrentAndNextFeeSchedule;
export const FeeSchedules = proto.CurrentAndNextFeeSchedule;

Reply:
I mean the SDK class name for `proto.CurrentAndNextFeeSchedule` should be called `FeeSchedules`. This does not mean you should update the proto exports, but rather the SDK type.